lib/embed.dart

```diff
@@ -413,6 +413,8 @@ class Embed extends EditorUi {
     var webOutputLabelElement = querySelector('#web-output-label');
     if (webOutputLabelElement != null) {
       webOutputLabel = DElement(webOutputLabelElement);
+    } else {
```

Comment:
nit: Is this necessary, since webOutputLabel will init to null?

Reply:
If we want to keep the `final` on `webOutputLabel`, then I believe so. As not yet being initialized when `late` is a bit different from having a `null` value. For example(https://dartpad.dev/?null_safety=true&id=3b6f35a9070221965a1ac79b577e14e5):

```dart
void main() {
  Test().init();
}

class Test {
  late final String? hi;
  
  void init() {
    if (12 != (24 ~/ 2)) { // Used to avoid dead code lint
      hi = 'hello';
    } else {
      //hi = null;
    }
    
    print(hi?.length); // You'll get a late initialization error here if you don't uncomment hi = null
  }
}
```

Reply:
That's it -- I missed that it was late.